frontend-react/src/components/Table/Table.tsx

```diff
@@ -124,10 +124,7 @@ const Table = ({
         return config.rows;
     }, [config.rows, filterManager?.sortSettings]);
 
-    const wrapperClasses = useMemo(
-        () => `grid-container margin-bottom-10 ${classes}`,
```

Comment:
If we want to be able to use atomic components anywhere, we can't tie them down to context-specific styles like the `grid-container` here -- that should be taken care of in an ancestor node, and Table/TableFilters should expand to their natural widths inside of that.